sdk/context.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2023 Mia srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sdk
+
+import (
+	"context"
+	"fmt"
+)
+
+type sdkKey struct{}
+
+func WithEvaluatorSDK(ctx context.Context, evaluator Evaluator) context.Context {
+	return context.WithValue(ctx, sdkKey{}, evaluator)
+}
+
+func GetEvaluatorSKD(ctx context.Context) (Evaluator, error) {
```

Comment:
```suggestion
func GetEvaluatorSDK(ctx context.Context) (Evaluator, error) {
```

If we keep SDK in the `With...` function this typo should definitely be fixed lol

Reply:
lol
Remove SKD